ffv1.md

```diff
@@ -351,28 +354,20 @@ JPEG2000-RCT is a Reversible Color Transform that codes RGB (red, green, blue) P
 SVGI:!---
 SVGI:![svg](rgb1.svg "rgb 1")
 SVGI:!---
-SVGC:rgb1.svg=$$\\\\begin{array}{ccccccc}Cb & = & b - g \\\\\\ Cr & = & r - g \\\\\\ Y & = & g + ( Cb + Cr)>>2 \\\\\\ g & = & Y - ( Cb + Cr ) >> 2 \\\\\\ r & = & Cr + g \\\\\\ b & = & Cb + g \\\\end{array}$$
+SVGC:rgb11.svg=$$\\\\begin{array}{ccccccc}Cb & = & b - g \\\\\\ Cr & = & r - g \\\\\\ Y & = & g + ( Cb + Cr)>>2 \\\\end{array}$$
```

Comment:
The name of the svg must match in the SVGI and SVGC section, ie match the filename of the svg in lines 355 and 357 as well as 364 and 366 as well as 397 and 399 as well as 406 and 408.